<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MosaicWizardPageForm</class>
 <widget class="QWizardPage" name="MosaicWizardPageForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mosaic</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_20">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_19">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_11">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>Type</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QComboBox" name="m_mosaicTypeComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Interpolator</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QComboBox" name="m_interpolatorTypeComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Blender</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QComboBox" name="m_blenderTypeComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="groupBox_8">
         <property name="title">
          <string>No Data</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="1" column="0">
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Value</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="m_noDataValueLineEdit"/>
              </item>
             </layout>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="m_noDataValueCheckBox">
              <property name="text">
               <string>Force Input Value</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>Options</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="1" column="0">
             <widget class="QCheckBox" name="m_autoEqualizeCheckBox">
              <property name="text">
               <string>Auto Equalize</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="m_rasterCacheCheckBox">
              <property name="text">
               <string>Use Raster Cache</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QStackedWidget" name="m_stackedWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>450</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_5">
        <layout class="QGridLayout" name="gridLayout_29">
         <item row="1" column="0">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Geo Mosaic</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_21">
            <item row="0" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>No Extra Parameters.</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::MinimumExpanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Tie Points Mosaic</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_28">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_27">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_24">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>Layer A</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QComboBox" name="m_tpmLayerAComboBox"/>
                </item>
               </layout>
              </item>
              <item row="1" column="0">
               <layout class="QGridLayout" name="gridLayout_25">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>Layer B</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QComboBox" name="m_tpmLayerBComboBox"/>
                </item>
               </layout>
              </item>
              <item row="2" column="0">
               <layout class="QGridLayout" name="gridLayout_23">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>Geometric Transformation</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QComboBox" name="m_tpmGeomTransformComboBox"/>
                </item>
               </layout>
              </item>
              <item row="3" column="0">
               <layout class="QGridLayout" name="gridLayout_22">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>Tie Points Link Type</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QComboBox" name="m_tpmLinkTypeComboBox"/>
                </item>
               </layout>
              </item>
              <item row="4" column="0">
               <layout class="QGridLayout" name="gridLayout_26">
                <item row="0" column="0">
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="0" column="1">
                 <widget class="QToolButton" name="m_tpmAcquireToolButton">
                  <property name="text">
                   <string>...</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item row="5" column="0">
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::MinimumExpanding</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Sequence Mosaic</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_16">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_15">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_17">
                <item row="0" column="0">
                 <layout class="QGridLayout" name="gridLayout_12">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>Geometric Transformation</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QComboBox" name="m_smGeomTransformComboBox"/>
                  </item>
                 </layout>
                </item>
                <item row="1" column="0">
                 <layout class="QGridLayout" name="gridLayout_13">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>Reference Band</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QComboBox" name="m_smRefBandComboBox"/>
                  </item>
                 </layout>
                </item>
                <item row="2" column="0">
                 <layout class="QGridLayout" name="gridLayout_14">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_4">
                    <property name="text">
                     <string>Mininumum Required Tie-Points Covered Area Percent</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QSpinBox" name="m_smMinTiePointsSpinBox">
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                    <property name="maximum">
                     <number>100</number>
                    </property>
                    <property name="value">
                     <number>25</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item row="1" column="0">
               <widget class="QGroupBox" name="m_tplpGroupBox">
                <property name="title">
                 <string>Tie Point Locator Parameters</string>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
                <layout class="QGridLayout" name="gridLayout_18">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item row="0" column="0">
                  <widget class="QWidget" name="m_tplpWidget" native="true"/>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_mosaicTypeComboBox</tabstop>
  <tabstop>m_interpolatorTypeComboBox</tabstop>
  <tabstop>m_blenderTypeComboBox</tabstop>
  <tabstop>m_noDataValueCheckBox</tabstop>
  <tabstop>m_noDataValueLineEdit</tabstop>
  <tabstop>m_rasterCacheCheckBox</tabstop>
  <tabstop>m_autoEqualizeCheckBox</tabstop>
  <tabstop>m_tpmLayerAComboBox</tabstop>
  <tabstop>m_tpmLayerBComboBox</tabstop>
  <tabstop>m_tpmGeomTransformComboBox</tabstop>
  <tabstop>m_tpmLinkTypeComboBox</tabstop>
  <tabstop>m_tpmAcquireToolButton</tabstop>
  <tabstop>m_smGeomTransformComboBox</tabstop>
  <tabstop>m_smRefBandComboBox</tabstop>
  <tabstop>m_smMinTiePointsSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_mosaicTypeComboBox</sender>
   <signal>activated(int)</signal>
   <receiver>m_stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
